src/makeInfinityAdder.js

```diff
@@ -7,6 +7,24 @@
 
 function makeInfinityAdder() {
   // write code here
+  let res = 0;
```

Comment:
if you use your res only in the if block, better solution will be to declare it there

Reply:
got it